pyomo/contrib/pynumero/cmake/asl_interface/src/AmplInterface.cpp

```diff
@@ -524,8 +529,13 @@ extern "C"
                               const_x, nx, const_lam, nc);
    }
 
-   void EXTERNAL_AmplInterface_free_memory(AmplInterface *p_ai) { 
+   void EXTERNAL_AmplInterface_free_memory(AmplInterface *p_ai) {
       p_ai->~AmplInterface();
    }
+
+   void EXTERNAL_AmplInterface_dummy(AmplInterface *p_ai) {
```

Comment:
Is this a debugging method that was left in?

Reply:
This is left there on purpose. The latest distribution of pynumero_libraries from conda-forge was not allowing for scaling the hessian matrix from the ASL interface. The dummy function was a mechanism to identified if scaling was enabled in the binaries or not. I believed I have updated the conda-forge recipe to now have this functionality as well. For the release I believe this can be kept and I will remove it in a future pull request, once is verified the binaries are working as expected. I will also remove the build rules for pynumero_SPARSE since now all sparse matrices are directly from scipy.